<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditDialog</class>
 <widget class="QDialog" name="EditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Recipe</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="editor_tab_widget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="general_tab">
      <attribute name="title">
       <string>&amp;General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="title_label">
           <property name="text">
            <string>&amp;Title:</string>
           </property>
           <property name="buddy">
            <cstring>title_edit</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="title_edit">
           <property name="maxLength">
            <number>60</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="categories_label">
           <property name="text">
            <string>&amp;Categories:</string>
           </property>
           <property name="buddy">
            <cstring>categories_edit</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="categories_edit"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="servings_label">
           <property name="text">
            <string>&amp;Yield:</string>
           </property>
           <property name="buddy">
            <cstring>servings_spin</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QSpinBox" name="servings_spin">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>9999</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="servings_unit_edit">
             <property name="maxLength">
              <number>40</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>164</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ingredients_tab">
      <attribute name="title">
       <string>&amp;Ingredients</string>
      </attribute>
      <widget class="QGroupBox" name="ingredient_group">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>210</y>
         <width>361</width>
         <height>81</height>
        </rect>
       </property>
       <property name="title">
        <string>I&amp;ngredient</string>
       </property>
      </widget>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>0</y>
         <width>372</width>
         <height>198</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QTreeView" name="ingredients_view"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="ingredient_group_button">
            <property name="text">
             <string>Add &amp;Group</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ingredient_button">
            <property name="text">
             <string>Add &amp;Ingredient</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="remove_button">
            <property name="text">
             <string>&amp;Remove</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ingredient_up_button">
            <property name="text">
             <string>&amp;Up</string>
            </property>
            <property name="icon">
             <iconset resource="anymeal.qrc">
              <normaloff>:/images/up.svg</normaloff>:/images/up.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ingredient_down_button">
            <property name="text">
             <string>&amp;Down</string>
            </property>
            <property name="icon">
             <iconset resource="anymeal.qrc">
              <normaloff>:/images/down.svg</normaloff>:/images/down.svg</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="instructions_tab">
      <attribute name="title">
       <string>In&amp;structions</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>editor_tab_widget</tabstop>
  <tabstop>title_edit</tabstop>
  <tabstop>categories_edit</tabstop>
  <tabstop>servings_spin</tabstop>
  <tabstop>servings_unit_edit</tabstop>
 </tabstops>
 <resources>
  <include location="anymeal.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>443</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>449</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
